paint/PAINT_ButtonFunctions.cpp

```diff
@@ -79,3 +107,13 @@ void paint::clearScreen(UIelement* control)
 	dw->clearScreen();
 }
 
+static ToggleButton* paint::CastToggleButton(UIelement* control)
```

Comment:
Whilst not technically wrong, static functions shouldn't reside in a namespace (other than the strange anonymous namespace:

```
namespace {
    // This is almost identical to static void foo()
   // I prefer static functions to anonymous namespace functions because the latter are name-mangled and so 
   // hard to find in the debugger
   void foo();
}
```

Drop the `paint` namespace.
**SHOULD**

Reply:
done